### PR TITLE
Migration data with Mapstore blob

### PR DESCRIPTION
**Pull request by hascar**
Migration data with population of mapstore tables

The tests are green here: http://jenkins.gem.lan/job/zdevel_oq-platform3/153/

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -38,27 +39,21 @@ pip install --default-timeout=100 Django==3.2.12
...
 # sudo rm oq-platform3/geoserver_data.tar.gz || true
 sudo rm -rf data || true
 sudo rm /usr/share/keyrings/docker-archive-keyring.gpg || true
 
 # display each command before executing it
 # set -x
 
-sudo apt-get -y update
-sudo apt-get -y upgrade
+# sudo apt-get -y update
```
Comment (nastasi-oq) [inline install.sh:51]: Delete if not needed.

File: install.sh [hunk 2]
```diff
@@ -81,60 +76,85 @@ inst_docker
...
+    sudo rm -rf geoserver
+fi
 
+# dump folder from ftp
+# wget https://ftp.openquake.org/oq-platform3/dump.tar.gz
+# tar zxf dump.tar.gz
+# sudo cp -pr dump $HOME/oq-platform3/openquakeplatform/data_commands/gs_data
+# rm -rf dump.tar.gz | true
+# rm -rf dump | true
+
```
Comment (nastasi-oq) [inline install.sh:157]: Delete it if not needed.

File: install.sh [hunk 3]
```diff
@@ -159,26 +178,21 @@ export COMPOSE_HTTP_TIMEOUT=400
...
 # import layers sql in db container and import in db postgres
 wget https://ftp.openquake.org/oq-platform3/sql_new.tar.gz
 tar zxf sql_new.tar.gz
 
-# for i in $(find . -name '/sql_new/*.sql'); do 
-#     sed -i 's/openquakeplatform_data/openquakeplatform/g' "$i"
-# done
+# mv sql/ghec_viewer_measure.sql sql/gheck_viewer_measure.sql.not
+# mv sql/isc_viewer_measure.sql sql/gheck_viewer_measure.sql.not
+# mv sql/isc_viewer_measure2.sql sql/gheck_viewer_measure.sql.not
```
Comment (nastasi-oq) [inline install.sh:195]: Delete it if not needed.

File: old/Dockerfile [hunk 4]
```diff
@@ -86,15 +86,15 @@ COPY isc_viewer /usr/src/{{project_name}}/isc_viewer
...
 
 # templates and static 
-COPY openquakeplatform/templates /usr/src/{{project_name}}/{{project_name}}/templates
-ADD openquakeplatform/templates/layers/templates/* /usr/src/geonode/geonode/layers/templates/layers/
-ADD openquakeplatform/templates/maps/templates/* /usr/src/geonode/geonode/maps/templates/maps/
-COPY openquakeplatform/static /usr/src/{{project_name}}/{{project_name}}/static
+# COPY openquakeplatform/templates /usr/src/{{project_name}}/{{project_name}}/templates
+# ADD openquakeplatform/templates/layers/templates/* /usr/src/geonode/geonode/layers/templates/layers/
+# ADD openquakeplatform/templates/maps/templates/* /usr/src/geonode/geonode/maps/templates/maps/
+# COPY openquakeplatform/static /usr/src/{{project_name}}/{{project_name}}/static
```
Comment (nastasi-oq) [inline old/Dockerfile:95]: Delete it if not needed.

File: openquakeplatform/.env.tmpl [hunk 5]
```diff
@@ -0,0 +1,239 @@
+COMPOSE_PROJECT_NAME=openquakeplatform
```
Comment (nastasi-oq) [inline openquakeplatform/.env.tmpl:1]: I suggest to rename `.env.tmpl` to `dot_env.tmpl` so it is visible in a normal `ls` command call.

File: openquakeplatform/.override_dev_env.sample [hunk 6]
```diff
@@ -0,0 +1,95 @@
+export DEBUG=True
```
Comment (nastasi-oq) [inline openquakeplatform/.override_dev_env.sample:1]: I suggest to rename `.override_dev_env.sample` to `dot_override_dev_env.sample` so it is visible in a normal `ls` command call.
Pay attention if it is used during tests.

File: openquakeplatform/fixtures/apikey_docker.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "fields": {
+            "user": 1000,
+            "key": "pyxW5djJ7XsjeFUXduAsGpR4xMGUwpeBGQRqTeT3",
```
Comment (nastasi-oq) [inline openquakeplatform/fixtures/apikey_docker.json:5]: Is it rebuilt during the deploy?

File: openquakeplatform/fixtures/sample_admin.json [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+		"date_joined": "2011-06-09 15:15:27",
+		"email": "ad@m.in",
+		"first_name": "",
+		"groups": [],
+		"is_active": true,
+		"is_staff": true,
+		"is_superuser": true,
+		"last_login": "2011-06-09 15:45:34",
+		"last_name": "",
+		"password": "pbkdf2_sha256$30000$rjuGt0Obn8on$cxF75frIOSaitNklLZ0IJ/VonUW0fwEFVF96o0M+lGc=",
```
[the remark this answers is not in the document]
Comment (hascar) [reply openquakeplatform/fixtures/sample_admin.json:12]: The data inside the database are different

File: openquakeplatform/fixtures/apikey_docker.json (hunk 7 above)
Comment (hascar) [reply openquakeplatform/fixtures/apikey_docker.json:5]: Doesn't exist inside the db

File: openquakeplatform/.env.tmpl (hunk 5 above)
Comment (hascar) [reply openquakeplatform/.env.tmpl:1]: Fixed with this commit
https://github.com/gem/oq-platform3/pull/21/commits/b0f2cfe1579ea74f04e61be55164a250cd2eb429

File: old/Dockerfile (hunk 4 above)
Comment (hascar) [reply old/Dockerfile]: Fixed with this commit 
https://github.com/gem/oq-platform3/pull/21/commits/b065525246d31d9dbb3af47a985db0238b0b1be7

File: install.sh (hunk 3 above)
Comment (hascar) [reply install.sh]: Fixed with this commit
https://github.com/gem/oq-platform3/pull/21/commits/82cb33fcb31be594dbd6b5dd952a703672b48ff1

File: install.sh (hunk 2 above)
Comment (hascar) [reply install.sh:156]: Re-activated in another branch

File: install.sh (hunk 1 above)
Comment (hascar) [reply install.sh]: Fixed with this commit
https://github.com/gem/oq-platform3/pull/21/commits/82cb33fcb31be594dbd6b5dd952a703672b48ff1

File: openquakeplatform/fixtures/default_oauth_apps_docker.json [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+        "skip_authorization": true,
+        "created": "2018-05-31T10:00:31.661Z",
+        "updated": "2018-05-31T11:30:31.245Z",
+        "algorithm": "RS256",
+        "redirect_uris": "http://geonode/geoserver",
+        "name": "GeoServer",
+        "authorization_grant_type": "authorization-code",
+        "client_type": "confidential",
+        "client_id": "Jrchz2oPY3akmzndmgUTYrs9gczlgoV20YPSvqaV",
+        "client_secret": "rCnp5txobUo83EpQEblM8fVj3QT5zb5qRfxNsuPzCqZaiRyIoxM4jdgMiZKFfePBHYXCLd7B8NlkfDBY9HKeIQPcy5Cp08KQNpRHQbjpLItDHv12GvkSeXp6OxaUETv3",
```
[the remark this answers is not in the document]
Comment (hascar) [reply openquakeplatform/fixtures/default_oauth_apps_docker.json:15]: Used here: https://github.com/GeoNode/geonode-project/tree/3.3.x/src/fixtures and not present inside the db

File: openquakeplatform/fixtures/default_oauth_apps.json [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+        "skip_authorization": true,
+        "created": "2018-05-31T10:00:31.661Z",
+        "updated": "2018-05-31T11:30:31.245Z",
+        "algorithm": "RS256",
+        "redirect_uris": "http://localhost:8080/geoserver/index.html\nhttp://localhost/geoserver/index.html",
+        "name": "GeoServer",
+        "authorization_grant_type": "authorization-code",
+        "client_type": "confidential",
+        "client_id": "Jrchz2oPY3akmzndmgUTYrs9gczlgoV20YPSvqaV",
+        "client_secret": "rCnp5txobUo83EpQEblM8fVj3QT5zb5qRfxNsuPzCqZaiRyIoxM4jdgMiZKFfePBHYXCLd7B8NlkfDBY9HKeIQPcy5Cp08KQNpRHQbjpLItDHv12GvkSeXp6OxaUETv3",
```
[the remark this answers is not in the document]
Comment (hascar) [reply openquakeplatform/fixtures/default_oauth_apps.json:15]: Used here: https://github.com/GeoNode/geonode-project/tree/3.3.x/src/fixtures and not present inside the db

**Review by nastasi-oq — APPROVED** [review]

**Pull request merged**